rocAL/rocAL/include/api/rocal_api_types.h

```diff
@@ -177,8 +177,8 @@ enum RocalTensorBackend {
  * currently supports following formats
  */
 enum class RocalROICordsType {
-    ROCAL_XYWH = 0,
-    ROCAL_LTRB = 1
+    ROCAL_LTRB = 0,
+    ROCAL_XYWH = 1
```

Comment:
@fiona-gladwin Why is this interchanged? Has this been reflected in all use cases?

Reply:
https://github.com/LakshmiKumar23/MIVisionX/pull/32#discussion_r1286480719
With respect to the above comment, We need to do a typecast from the RocalROIType to the RpptROIType directly
Hence the order should be the same as that of RPP. Hence this change is made